src/MultiReport/Keyboard.cpp

```diff
@@ -139,6 +139,21 @@ int Keyboard_::sendReportUnchecked(void) {
 
 
 int Keyboard_::sendReport(void) {
+  // ChromeOS 51-60 (at least) bug: if a modifier and a normal keycode are added in the
+  // same report, in some cases the shift is not applied (e.g. `shift`+`[` doesn't yield
+  // `{`). To compensate for this, check to see if the modifier byte has changed. If so,
+  // check to see if any modifier keys have been pressed (probably not necessary to do
+  // this step, actually). If so, copy the modifier byte to the previous key report, and
+  // resend it before proceeding.
+  uint8_t mods_changed = lastKeyReport.modifiers ^ keyReport.modifiers;
```

Comment:
I do enjoy playing with bitwise operators, but alas, it's totally unnecessary here.